spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/actions/BaseExpireSnapshotsSparkAction.java

```diff
@@ -70,12 +70,13 @@
     extends BaseSparkAction<ExpireSnapshots, ExpireSnapshots.Result> implements ExpireSnapshots {
   private static final Logger LOG = LoggerFactory.getLogger(BaseExpireSnapshotsSparkAction.class);
 
+  // Make this variable public to be referenced for procedure option update
```

Comment:
I don't think this comment is really necessary, I think it's fine to just let it be a public static final string

Reply:
+1

Reply:
Actually, what about moving this constant to `ExpireSnapshots`? What do you think, @RussellSpitzer? We do that in the action for rewriting data files, for instance.

Reply:
I think putting the constant into `ExpireSnapshots` might be a good idea, and also I noticed that actions are all base class, e.g. `BaseExpireSnapshotsSparkAction`. Does this mean that the behavior can be overwritten in future?